media/css/firefox/os/get_device.less

```diff
@@ -97,102 +102,95 @@
+    .o2 {
+        background: transparent url('/media/img/firefox/os/logo/o2.png') top center no-repeat;
     }
-    .grameenphone {
-        background: transparent url('/media/img/firefox/os/logo/grameenphone.gif') top center no-repeat;
+    .orange {
+        background: transparent url('/media/img/firefox/os/logo/orange.png') top center no-repeat;
+    }
+    .rue-du-commerce {
+        background: transparent url('/media/img/firefox/os/logo/rue-du-commerce.jpg') top center no-repeat;
     }
     .spice {
         background: transparent url('/media/img/firefox/os/logo/spice.gif') top center no-repeat;
     }
-    .flipkart {
-        background: transparent url('/media/img/firefox/os/logo/flipkart.png') top center no-repeat;
-    }
-    .jbhifi {
-        background: transparent url('/media/img/firefox/os/logo/jbhifi.png') top center no-repeat;
+    .t-mobile {
+        background: transparent url('/media/img/firefox/os/logo/t-mobile.jpg') top center no-repeat;
     }
-    .zen {
-        background: transparent url('/media/img/firefox/os/logo/zen.png') top center no-repeat;
+    .telcel {
+        background: transparent url('/media/img/firefox/os/logo/telcel.jpg') top center no-repeat;
     }
-    .mistergooddeal {
-        background: transparent url('/media/img/firefox/os/logo/mistergooddeal.jpg') top center no-repeat;
+    .telenor {
+        background: transparent url('/media/img/firefox/os/logo/telenor.jpg') top center no-repeat;
     }
     .topachat {
         background: transparent url('/media/img/firefox/os/logo/topachat.png') top center no-repeat;
     }
+    .webdistrib {
+        background: transparent url('/media/img/firefox/os/logo/webdistrib.jpg') top center no-repeat;
+    }
+    .zte-france {
+        background: transparent url('/media/img/firefox/os/logo/zte.png') top center no-repeat;
+    }
+
+    /*
+    providers below are currently unused, but may be used again in the future
+
     .cherry-mobile {
         background: transparent url('/media/img/firefox/os/logo/cherry-mobile.png') top center no-repeat;
     }
-    .megafon {
-        background: transparent url('/media/img/firefox/os/logo/megafon.png') top center no-repeat;
+    .flipkart {
+        background: transparent url('/media/img/firefox/os/logo/flipkart.png') top center no-repeat;
     }
-    .au {
-        background: transparent url('/media/img/firefox/os/logo/au.png') top center no-repeat;
+    .grameenphone {
+        background: transparent url('/media/img/firefox/os/logo/grameenphone.gif') top center no-repeat;
+    }
+    .intex {
+        background: transparent url('/media/img/firefox/os/logo/intex.png') top center no-repeat;
+    }
+    .jbhifi {
+        background: transparent url('/media/img/firefox/os/logo/jbhifi.png') top center no-repeat;
     }
     .mtn {
         background: transparent url('/media/img/firefox/os/logo/mtn.png') top center no-repeat;
     }
```

Comment:
Well, none of the providers in the commented out list are currently active, but I figured it best to keep them around for easy re-activation should they be needed. It would be a shame if we got an MTN link in a couple months and had to re-create the class.

Or are you saying MTN is no longer in the cell service/phone business?